src/modules.py

```diff
@@ -447,7 +451,236 @@ def parse_panda(config: Dict[str, Any]) -> IocCollection:
     for cnc in config.get("cnc", []):
         iocs.try_add_url(cnc["url"])
     if "comm_public_key" in config:
-        iocs.try_add_rsa_from_pem(config["comm_public_key"])
+        if type(config["comm_public_key"]) == str:
+            iocs.try_add_rsa_from_pem(config["comm_public_key"])
+        elif type(config["comm_public_key"]) == dict:
+            key = config["comm_public_key"]
+            iocs.add_rsa_key(RsaKey(int(key["n"]), int(key["e"])))
+
     if "public_key" in config:
-        iocs.try_add_rsa_from_pem(config["public_key"])
+        if type(config["public_key"]) == str:
+            iocs.try_add_rsa_from_pem(config["public_key"])
+        elif type(config["public_key"]) == dict:
+            key = config["public_key"]
+            iocs.add_rsa_key(RsaKey(int(key["n"]), int(key["e"])))
+
+    return iocs
+
+
+@module("vjworm")
+def parse_vjworm(config: Dict[str, Any]) -> IocCollection:
+    iocs = IocCollection()
+    for urls in config.get("urls", []):
+        iocs.try_add_url(urls["url"])
+    return iocs
+
+
+@module("nymaim")
+def parse_nymaim(config: Dict[str, Any]) -> IocCollection:
+    iocs = IocCollection()
+    if "encryption_key" in config:
+        iocs.add_key("encryption_key", config["encryption_key"])
```

Comment:
key type should be `"other"`, not `"encryption_key"`

(It's not a formal enum, but if we don't know anything specific about the encyption key it should be saved as "other". Actually, IIRC it's serpent key, but let's stick to `"other"`